app/src/main/res/values/strings.xml

```diff
@@ -150,6 +165,24 @@
     <string name="identification_text">Show us who you are\.\.\.</string>
     <string name="start_planting">Start Planting</string>
 
-    <string name="build_version_title" translatable="false">Build version  </string>
-    <string name="tree_tracker_title" translatable="false">Tree Tracker(debug) </string>
+    <string name="build_version_title" translatable="false">Build version</string>
+    <string name="tree_tracker_title" translatable="false">Tree Tracker(debug)</string>
+
+    <!--SignUp screen strings-->
+    <string name="greenstand_welcome_text" translatable="false">Greenstand Tree Tracker</string>
```

Comment:
Is the signup screen only supposed to be in english? I see `translatable = false` on all strings